### PR TITLE
Update ember to 2.8, update other dependencies and CI tool

### PR DESCRIPTION
**Pull request by dafortin**
#major# 

# CHANGELOG
- **Update** to ember@2.8
- **Update** all other dependencies
- **Update** CI tools (linting, travis, etc.)
- **Add** github templates
- **Fix** linting issues
- **Fix** readme badges
- **Update** build to run tests on Chrome

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,43 +23,48 @@
...
     "ember-code-snippet": "1.4.0",
-    "ember-computed-decorators": "0.2.2",
-    "ember-concurrency": "0.7.10",
+    "ember-computed-decorators": "~0.2.0",
+    "ember-concurrency": "~0.7.0",
     "ember-disable-proxy-controllers": "^1.0.1",
     "ember-elsewhere": "0.3.0",
-    "ember-engines": "0.2.4",
-    "ember-export-application-global": "^1.0.4",
+    "ember-engines": "~0.3.4",
```
Comment (rox163) [inline package.json]: does the dummy app still launch the test engine after this upgrade? iirc the test engine is still at 0.2.x

File: package.json [hunk 2]
```diff
@@ -20,43 +23,48 @@
...
+    "ember-export-application-global": "^1.0.5",
     "ember-frost-core": "0.29.0",
-    "ember-hook": "^1.3.1",
-    "ember-load-initializers": "^0.5.0",
+    "ember-hook": "^1.0.0",
```
Comment (rox163) [inline package.json]: why are we moving backwards here for minimum supported version? The fixes we need are in 1.3.1 at least

File: package.json (hunk 1 above)
Comment (dafortin) [inline package.json]: The project was giving me errors with 0.2.x. I did update it and now it looks like it's not able to submit new item. Everything is working except creating a new item.

File: package.json [hunk 3]
```diff
@@ -20,43 +23,48 @@
   "author": "",
```
Comment (rox163) [inline package.json]: should update the node version since this PR is still open. >=6.0.0

File: .travis.yml [hunk 4]
```diff
@@ -1,8 +1,9 @@
+sudo: required
+dist: trusty
 language: node_js
 node_js:
 - '5.0'
```
Comment (rox163) [inline .travis.yml]: update to 6.9

**Comment by rox163** [comment]
:+1:

**Pull request merged**